addons/mail/static/src/discuss/call/common/call.scss

```diff
@@ -10,11 +10,22 @@
     &.o-minimized {
         height: 20%; // ensures that the view returns to the right height when resized
         min-height: #{"max(20%, 130px)"};
```

Comment:
This can be removed, so that chat window height in non-collapsible mode takes 50% or chat window area like before.